### PR TITLE
Added a lot more dependencies to the RHEL6 comp

### PR DESCRIPTION
**Pull request by skottler**

### CODE REVIEW

File: extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml [hunk 1]
```diff
@@ -8,77 +8,112 @@
...
+      <packagereq type="default">ruby193-rubygem-apipie-rails</packagereq>
+      <packagereq type="default">ruby193-rubygem-arel</packagereq>
+      <packagereq type="default">ruby193-rubygem-audited-activerecord</packagereq>
+      <packagereq type="default">ruby193-rubygem-audited</packagereq>
+      <packagereq type="default">ruby193-rubygem-augeas</packagereq>
+      <packagereq type="default">ruby193-rubygem-awesome_print</packagereq>
+      <packagereq type="default">ruby193-rubygem-bigdecimal</packagereq>
+      <packagereq type="default">ruby193-rubygem-builder</packagereq>
+      <packagereq type="default">ruby193-rubygem-bundler</packagereq>
+      <packagereq type="default">ruby193-rubygem-bundler</packagereq>
```
Comment (domcleal) [inline extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml]: Dupe?

File: extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml [hunk 2]
```diff
@@ -8,77 +8,112 @@
...
+      <packagereq type="default">ruby193-rubygem-fog</packagereq>
+      <packagereq type="default">ruby193-rubygem-foremancli</packagereq>
+      <packagereq type="default">ruby193-rubygem-formatador</packagereq>
+      <packagereq type="default">ruby193-rubygem-gettext_i18n_rails</packagereq>
+      <packagereq type="default">ruby193-rubygem-hirb-unicode</packagereq>
+      <packagereq type="default">ruby193-rubygem-hirb</packagereq>
+      <packagereq type="default">ruby193-rubygem-i18n</packagereq>
+      <packagereq type="default">ruby193-rubygem-i18n_data</packagereq>
+      <packagereq type="default">ruby193-rubygem-journey</packagereq>
+      <packagereq type="default">ruby193-rubygem-journey</packagereq>
```
Comment (domcleal) [inline extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml]: Dupe

File: extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml [hunk 3]
```diff
@@ -8,77 +8,112 @@
...
+      <packagereq type="default">ruby193-rubygem-jquery-rails</packagereq>
+      <packagereq type="default">ruby193-rubygem-jquery-ui-rails</packagereq>
+      <packagereq type="default">ruby193-rubygem-json</packagereq>
+      <packagereq type="default">ruby193-rubygem-mail</packagereq>
+      <packagereq type="default">ruby193-rubygem-mail</packagereq>
```
Comment (domcleal) [inline extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml]: Dupe

File: extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml [hunk 4]
```diff
@@ -8,77 +8,112 @@
...
+      <packagereq type="default">ruby193-rubygem-twitter-bootstrap-rails</packagereq>
+      <packagereq type="default">ruby193-rubygem-tzinfo</packagereq>
+      <packagereq type="default">ruby193-rubygem-uglifier</packagereq>
+      <packagereq type="default">ruby193-rubygem-unicode-display_width</packagereq>
+      <packagereq type="default">ruby193-rubygem-uuidtools</packagereq>
+      <packagereq type="default">ruby193-rubygem-virt</packagereq>
+      <packagereq type="default">ruby193-rubygem-will_paginate</packagereq>
+      <packagereq type="default">ruby193-rubygem-wirb</packagereq>
+      <packagereq type="default">ruby193-rubygems</packagereq>
+      <packagereq type="default">ruby193-v8</packagereq>
```
Comment (domcleal) [inline extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml:112]: Why's v8 part of the Ruby stack?

Comment (skottler) [inline extras/packaging/rpm/rel-eng/comps/comps-foreman-rhel6.xml:112]: That's just the package's name. It's really just v8.

**Comment by skottler** [comment]
Removed the dupes.

**Comment by domcleal** [comment]
Merged as 48a9cf0, thanks @skottler!

**Pull request closed without merging**